doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -181,17 +181,23 @@ To import the appliance:
 ]
 ----
 +
-. Use AWS-CLI tools to import the image. Run the following command:
+. Use the AWS-CLI tools to import a disk as a snapshot. See the
+link://docs.aws.amazon.com/AWS[AWS documentation] on using VM Import/Export to
+import a disk as a snapshot.
 +
 ----
-$ aws ec2 import-image --disk-containers file://containers.json --region BUCKET_REGION
+$ aws ec2 import-snapshot --disk-containers file://containers.json --region BUCKET_REGION
```

Comment:
Please remove `--region BUCKET_REGION` from the command as`region` isn't a valid option (and it wasn't even with `import-image`). Thanks!

Reply:
I removed --region BUCKET_REGION, as pointed out above.

I corrected the link to the AWS landing page, as noted above -- removed the final "/AWS"